app/controllers/conferences_controller.rb

```diff
@@ -16,12 +16,11 @@ def show
 
   def create
     team = current_user.student_team
```

Comment:
Okay, I see what you mean! Do you mind if we set this on hold, then? And leave it in this separate PR. 
I am sorry for the complications :-(
But let's first _implement_ the enhancement before we solve bugs in that enhancement 😂